browserid-verifier - docker build failed

docker build failed on Debian Stretch.

see paste.
https://paste.debian.net/1040699/

Pulling context out of the pastebin, this appears to be a failure to build the bignum dependency:

```
> bignum@0.12.5 install /app/node_modules/bignum
> node-pre-gyp install --fallback-to-build

[91mnode-pre-gyp ERR! Tried to download(403): https://rvagg-node.s3-us-west-2.amazonaws.com/bignum/v0.12.5/bignum-v0.12.5-node-v64-linux-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for bignum@0.12.5 and node@10.9.0 (node-v64 ABI, musl) (falling back to source compile with node-gyp) 
[0m[91mnode-pre-gyp ERR! Tried to download(undefined): https://rvagg-node.s3-us-west-2.amazonaws.com/bignum/v0.12.5/bignum-v0.12.5-node-v64-linux-x64.tar.gz 
node-pre-gyp ERR! Pre-built binaries not found for bignum@0.12.5 and node@10.9.0 (node-v64 ABI, musl) (falling back to source compile with node-gyp) 
[0mmake: Entering directory '/app/node_modules/bignum/build'
  ACTION Regenerating Makefile
  CXX(target) Release/obj.target/bignum/bignum.o
make: Entering directory '/app/node_modules/bignum/build'
  CXX(target) Release/obj.target/bignum/bignum.o
[91m../bignum.cc: In function 'int BN_jacobi_priv(const BIGNUM*, const BIGNUM*, int*, BN_CTX*)':
../bignum.cc:143:6: error: expected '(' before 'BN_is_zero'
   if BN_is_zero(a1) /* step 1 */
      ^~~~~~~~~~
```

It's pinned to v0.12.5 in `npm-shrinkwrapjson`, but there's a v0.13.0 that was released recently.  We could try updating to that.

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "license": "MPL-2.0",
   "name": "browserid-verifier",
   "description": "A node.js verification server for BrowserID assertions.",
-  "version": "0.9.5",
+  "version": "0.9.6",
   "repository": {
     "type": "git",
     "url": "https://github.com/mozilla/browserid-verifier"

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
   "dependencies": {
     "async": "2.0.1",
     "body-parser": "1.18.2",
-    "browserid-local-verify": "0.5.1",
+    "browserid-local-verify": "0.5.2",
     "compute-cluster": "0.0.9",
     "convict": "1.5.0",
     "express": "4.15.5",

